velox/functions/sparksql/coverage/README.md

```diff
@@ -0,0 +1,23 @@
+# Coverage Maps
+
+Velox includes a subset of scalar and aggregate Spark functions.
+velox_sparksql_coverage utility is used to generate a list of available
+functions. The output uses reStructured text format suitable for use in
+Velox documentation.
+
+> velox_sparksql_coverage
+
+Generates a list of Spark functions available in Velox. The output
+to be copy-pasted into velox/docs/spark_functions.rst file.
+
+> velox_sparksql_coverage --all
+
+Generates coverage map using all Spark functions. The output to be copy-pasted
```

Comment:
It would be nice to automate this process and add it to the nightly doc-gen job. CC: @kgpai

Reply:
Hi Masha, Sure I will do that; Also As I pointed out in a comment above, it should be possible to have this generated via sphinx code automatically without the need for a separate process via pyVelox. It doesnt have to be done in this PR, but we should aim to move such utility code to PyVelox.